sag_et.par

```diff
@@ -4,195 +4,241 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/||\
 n 7 4
+/||\

```

Comment:
This was the only spot in this config file where an ability was being leveraged to have multiple EDOs share the same symbol set. I wasn't sure why it wasn't being leveraged more. I could understand if every other place it otherwise could be used it couldn't be used just because the EDOs sharing symbols differed on subsets; however that does not seem to be the case (e.g. 31 and 38, neither with any subsets but both with the same symbol set, could perfectly well have been grouped, but they were not).

I briefly considered leveraging it everywhere it could be leveraged. However, I decided that including that change at this time would obscure the more important changes I am suggesting. Also that would have been a lot of work.

In the end I went the other direction, breaking up this one and only place where this ability was being leveraged. I think the n, d, s labels in this file are challenging enough to get one's head around without adding that layer of complexity.